src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -9,18 +9,20 @@ import { StoreMixin } from "mesosphere-shared-reactjs";
 import { Dropdown, Tooltip, Modal } from "reactjs-components";
 
 import Icon from "#SRC/js/components/Icon";
-import Breadcrumb from "../../components/Breadcrumb";
-import BreadcrumbTextContent from "../../components/BreadcrumbTextContent";
-import CosmosPackagesStore from "../../stores/CosmosPackagesStore";
+import Breadcrumb from "#SRC/js/components/Breadcrumb";
+import BreadcrumbTextContent from "#SRC/js/components/BreadcrumbTextContent";
+import CosmosPackagesStore from "#SRC/js/stores/CosmosPackagesStore";
 import defaultServiceImage
-  from "../../../../plugins/services/src/img/icon-service-default-large@2x.png";
-import Image from "../../components/Image";
-import ImageViewer from "../../components/ImageViewer";
-import Loader from "../../components/Loader";
-import MetadataStore from "../../stores/MetadataStore";
-import Page from "../../components/Page";
-import RequestErrorMsg from "../../components/RequestErrorMsg";
-import StringUtil from "../../utils/StringUtil";
+  from "#PLUGINS/services/src/img/icon-service-default-large@2x.png";
+import Image from "#SRC/js/components/Image";
+import ImageViewer from "#SRC/js/components/ImageViewer";
+import Loader from "#SRC/js/components/Loader";
+import MetadataStore from "#SRC/js/stores/MetadataStore";
+import Page from "#SRC/js/components/Page";
+import RequestErrorMsg from "#SRC/js/components/RequestErrorMsg";
+import StringUtil from "#SRC/js/utils/StringUtil";
+
+const semver = require("semver");
```

Comment:
I think import would be better and more consistent (for that reason also on the top)
Sorry I should have caught this in the review process, I think we need to leave it there for now, right?

Reply:
I agree that we should have catched that on the normal PR. Now that it made in I would leave it :/

Reply:
@juliangieseke could you pleas file a JIRA for this one?